fjord/base/translation.py

```diff
@@ -0,0 +1,6 @@
+from django.utils.functional import lazy
+from django.utils.translation import ugettext, ungettext
+
+
+ugettext_lazy = lazy(ugettext, unicode)
+ungettext_lazy = lazy(ungettext, unicode)
```

Comment:
It looks like django has lazy versions already:

https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#lazy-translation

Does that not work for what we're doing?